indico/modules/rb/__init__.py

```diff
@@ -27,6 +28,7 @@
 
 rb_settings = SettingsProxy('roombooking', {
     'managers_edit_rooms': False,
+    'hidden_from_unauthorised_users': False,
```

Comment:
We use en_US in code, so `unauthorized` with `z`.

Naming is hard, and IMHO `hide_module_if_unauthorized` would be better. I'm still not 100% happy with that name either, but it seems clearer since otherwise the name could also be understood as applying to individual rooms.

Reply:
Okay, this is much clearer than before. Yes indeed naming is hard.